### PR TITLE
Add pyftdi recipe

### PR DESCRIPTION
**Pull request by abloemert**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pyftdi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  sha256: 3b5fedfc81fc195fcc14bfa45aefaca0cd7a98f563aa0fcf60a8a9cdc18ea62d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/pyftdi/meta.yaml]:
```suggestion
    - python >=3.5
```

**Comment by marcelotrevisani** [comment]
Is it still a draft or do you want to merge it?

**Comment by abloemert** [comment]
I'd like to merge it. Any idea why the AppVeyor build is failing?

**Comment by scopatz** [comment]
noarch packages only need to run on linux. Thanks!

**Pull request merged**